banyanmonitorgui/banyan_monitor_gui.py

```diff
@@ -279,7 +277,7 @@ def clean_up(self):
 
 def start_gui():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-b", dest="back_plane_ip_address", default="127.0.0.1",
```

Comment:
Changed this to None so that the base class can figure out the actual IP address of the local computer.